lib/_stream_writable.js

```diff
@@ -202,6 +192,12 @@ ObjectDefineProperty(WritableState.prototype, 'buffer', {
      'instead.', 'DEP0003')
 });
 
+ObjectDefineProperty(WritableState.prototype, 'bufferedRequestCount', {
```

Comment:
Should `bufferedRequestCount` be documented?

Reply:
Not in my opinion. It's here for backwards compat.

Reply:
Oh, I missed that. I agree that there is no need to document this then.